src/multiple.ts

```diff
@@ -0,0 +1,13 @@
+import { Expression } from "./types";
+import { simplifyAndGroup } from "./utils";
+
+export default function multiple(
+  input: Expression,
+  minTimes?: number,
+  maxTimes?: number
+) {
+  const output = minTimes
```

Comment:
Let's try to support:

- `multiple('foo')` — `foo*`
- `multiple('foo', 2)` — `foo{2}`
- `multiple('foo', 2, 5)` — `foo{2, 5}`
- `multiple('foo', 2, Infinity)` — `foo{2,}`